add methods for removing hotkeys and hotstrings (#215)
spyoungtech/ahk: docs/README.md

ahk.add_hotstring('btw', my_callback) # call python function in response to the hotstring
```

You can also remove hotstrings:

```python

> ahk.remove_hotstring('btw')  # remove hotkey by the trigger sequence

Author: ```suggestion
ahk.remove_hotstring('btw')  # remove a hotstring by its trigger sequence
```
